### PR TITLE
Convert the editor and default theme fonts to WOFF2 format to save space.

### PR DESCRIPTION
**Pull request by bruvzg**
Makes an editor executable about 5.2 MB smaller. I was not able to notice any change in loading speed, or any visual differences.

### CODE REVIEW

File: COPYRIGHT.txt [hunk 1]
```diff
@@ -166,18 +166,18 @@ Comment: etcpak
 Copyright: 2013-2021, Bartosz Taudul
 License: BSD-3-clause
 
-Files: ./thirdparty/fonts/DroidSans*.ttf
-Comment: DroidSans font
+Files: ./thirdparty/fonts/DroidSans*.woff2
+Comment: DroidSans font, converted from .ttf
```
Comment (akien-mga) [inline COPYRIGHT.txt:170]: I wouldn't bother specifying this in COPYRIGHT.txt, that doesn't sound like a change that warrants being mentioned in this context.

File: thirdparty/README.md [hunk 2]
```diff
@@ -130,25 +130,25 @@ Files extracted from upstream source:
...
 
-- `NotoSans*.ttf`, `NotoNaskhArabicUI_*.ttf`:
+- `NotoSans*.woff2`, `NotoNaskhArabicUI_*.woff2`:
   * Upstream: https://github.com/googlefonts/noto-fonts
   * Version: v2017-10-24-phase3-second-cleanup
   * License: OFL-1.1
   * Comment: Use UI font variant if available, because it has tight vertical metrics and
     good for UI.
-- `JetBrainsMono_Regular.ttf`:
+- `JetBrainsMono_Regular.woff2`:
```
Comment (akien-mga) [inline thirdparty/README.md:139]: I don't know if that makes any significant difference, but JetBrains seem to provide a woff2 version of the font themselves: https://github.com/JetBrains/JetBrainsMono/blob/master/fonts/webfonts/JetBrainsMono-Regular.woff2

Comment (bruvzg) [reply thirdparty/README.md:139]: It's probably not, WOFF is just a compressed wrapper, all font structures should stay the same. Their version is 3 KB larger, but it probably depends only on a tool that was used to compress it.

File: COPYRIGHT.txt (hunk 1 above)
Comment (bruvzg) [reply COPYRIGHT.txt]: OK, removed.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**